src/pages/commits/CommitsWrapper.tsx

```diff
@@ -28,91 +36,130 @@ export const CommitsWrapper: React.FC<Props> = ({
   chartType,
   hasTaskFilter,
   hasFilters,
+  onChangeChartType,
 }) => {
+  const versionToGroupedTaskStatsMap = useMemo(() => {
+    if (versions) {
+      return getAllTaskStatsGroupedByColor(versions);
+    }
+    return undefined;
+  }, [versions]);
+
+  const maxGroupedTaskStats = useMemo(() => {
+    if (versionToGroupedTaskStatsMap) {
+      return findMaxGroupedTaskStats(versionToGroupedTaskStatsMap);
+    }
+    return undefined;
+  }, [versionToGroupedTaskStatsMap]);
```

Comment:
are the `return undefined`s needed here? I removed them and observed the same behavior with or without, but I might have overlooked something